console/account/how-to/close-account.mdx

```diff
@@ -7,7 +7,7 @@ content:
   paragraph: This page explains how to close an account
```

Comment:
Is there a difference between closing and deleting an account? For me "closing" could also be understood as "deactivating", whereas "deleting" implies that the account is gone forever.

Reply:
Also, we should unify the terms, as in the title we use "close" and in the steps we use "deletion" and "erase".

Reply:
I think it should be in accordance with the wording in the FAQ too https://www.scaleway.com/en/docs/faq/account/#i-would-like-to-delete-my-scaleway-account-and-data